fmriprep/workflows/bold/t2s.py

```diff
@@ -74,26 +70,23 @@ def init_bold_t2s_wf(echo_times, mem_gb, omp_nthreads,
 used to fit the model.
 The calculated T2* map was then used to optimally combine preprocessed BOLD across
 echoes following the method described in [@posse_t2s].
-The optimally combined time series was carried forward as the *preprocessed BOLD*{}.
-""".format('' if not t2s_coreg else ', and the T2* map was also retained as the BOLD reference')
+The optimally combined time series was carried forward as the *preprocessed BOLD*.
+"""
 
     inputnode = pe.Node(niu.IdentityInterface(fields=['bold_file']), name='inputnode')
 
-    outputnode = pe.Node(niu.IdentityInterface(fields=['bold', 'bold_mask', 'bold_ref_brain']),
+    outputnode = pe.Node(niu.IdentityInterface(fields=['bold', 'bold_ref_brain']),
                          name='outputnode')
 
     LOGGER.log(25, 'Generating T2* map and optimally combined ME-EPI time series.')
 
     t2smap_node = pe.Node(T2SMap(echo_times=echo_times), name='t2smap_node')
-    skullstrip_t2smap_wf = init_skullstrip_bold_wf(name='skullstrip_t2smap_wf')
 
     workflow.connect([
         (inputnode, t2smap_node, [('bold_file', 'in_files')]),
-        (t2smap_node, outputnode, [('optimal_comb', 'bold')]),
-        (t2smap_node, skullstrip_t2smap_wf, [('t2star_map', 'inputnode.in_file')]),
-        (skullstrip_t2smap_wf, outputnode, [
-            ('outputnode.mask_file', 'bold_mask'),
-            ('outputnode.skull_stripped_file', 'bold_ref_brain')]),
+        (t2smap_node, outputnode, [
+            ('optimal_comb', 'bold'),
+            ('t2star_map', 'bold_ref_brain')]),
```

Comment:
This workflow is now `inputnode -> t2smap_node -> outputnode`, so the workflow itself can be replaced in the calling function by `t2smap_node`. Unless it's useful to keep this space open to expand the workflow in the future?

Reply:
I don't know if implementing it would require that `bold_t2s_wf` remain a workflow, but in https://github.com/poldracklab/fmriprep/issues/1010#issuecomment-619209589 I'm proposing a way to incorporate tedana's component classification into fMRIPrep. It might be worth it to keep the workflow around until #1010 is resolved at least.

I can remove `bold_ref_brain` now.

Reply:
Sure. Let's keep this change minimal and we can reconsider architecture later. I realized there's also the workflow description that would need to be moved out of this separate module.